### PR TITLE
Complementing End Game contents

### LINKED ISSUE #201: More side activities separate from the storyline.
More activities to do apart from doing the main storyline. Ex: What to do after the second AirWizard is defeated.

Suggestions: Addition of End Game items and equipment. Ex: The second AirWizard will drop materials to craft a new pickaxe. This new Pickaxe can mine Cloud Ore and thus giving Cloud Ore a use.

### PR DESCRIPTION
**Pull request by BenCheung0422**
Resolves #201

According to #201, a new floor has been "added" in #403;
For other additions,
Adds new mobs, e.g. chickens, spiders?
Adds tame-able pets, i.e. dogs/wolfs, cats?
Adds End Game equipment made from cloud ore?
More would be discussed later.
Implements "curse"? Details: https://github.com/MinicraftPlus/minicraft-plus-revived/pull/440#issuecomment-1449886011.

### CODE REVIEW

File: src/main/java/minicraft/entity/mob/Sheep.java [hunk 1]
```diff
@@ -18,7 +19,34 @@ public class Sheep extends PassiveMob {
...
+		LIGHT_GRAY(0x1_9D9D97),
+		GRAY(0x1_474F52),
+		PINK(0x1_F38BAA),
+		LIME(0x1_80C71F),
+		YELLOW(0x1_FED83D),
+		LIGHT_BLUE(0x1_3AB3DA),
+		MAGENTA(0x1_C74EBD),
+		ORANGE(0x1_F9801D),
+		WHITE(0x1_F9FFFE),
+		;
```
Comment (Makkkkus) [inline src/main/java/minicraft/entity/mob/Sheep.java:39]: Semicolons should never be on a line by themselves.

**Comment by BenCheung0422** [comment]
Colored sheep is moved to #445.

**Comment by Makkkkus** [comment]
I'm closing this because of lacking activity.

**Comment by Makkkkus** [comment]
Keep to the roadmap.

**Comment by BenCheung0422** [comment]
This is related to expanding the storyline.

**Comment by BenCheung0422** [comment]
Isn't this related?

**Comment by Makkkkus** [comment]
It is not, no. The story is set before the end of the game, not after it.

**Comment by BenCheung0422** [comment]
Therefore, this is expanding the storyline.

**Comment by Makkkkus** [comment]
No, we're not adding new mobs. It doesn't add anything new to the story.

**Comment by BenCheung0422** [comment]
But I think "curse" is a big addition to the story.

**Comment by Makkkkus** [comment]
Make that its own pull request, then. I doubt the implementation will be good, though.

**Comment by BenCheung0422** [comment]
New mobs should also be added for the implementation and completion to the "curse" implementation.

**Pull request closed without merging**